Cargo.toml

```diff
@@ -1,7 +1,17 @@
 [package]
-name = "derive-restricted"
+name = "derive-where"
 version = "0.1.0"
 edition = "2021"
+# rust-version
+description = "Deriving with custom trait bounds"
+documentation = "https://docs.rs/derive-where"
+readme = "README.md"
+homepage = "https://github.com/ModProg/derive-where"
+repository = "https://github.com/ModProg/derive-where"
+license = "MIT OR Apache-2.0"
+# keywords
+# categories
+# include
```

Comment:
I don't think we need the `include` tag, but we might want to use the `exclude` tag for the up and coming `.github` folder!
```suggestion
# exclude
```

Reply:
yeah, I would have just added something like `clap`:
```toml
include = [
	"src/**/*",
	"Cargo.toml",
	"LICENSE-*",
	"README.md"
]
```
Examples and tests don't need to be in there as well IIRC

Reply:
For a moment I thought that's neat, better to control whats included then trying to exclude a bunch of things. But I can't find the documentation actually saying anything like that: https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields. So I think we will still need the `exclude` field, unless I'm missing something here.

I would like to know why `clap` does this, maybe they wrongly thought what I assumed here too?

Reply:
> The options are mutually exclusive; setting include will override an exclude. If you need to have exclusions to a set of include files, use the ! operator described below.

Reply:
This doesn't tell me that using `include` will **only** include the files specified. I will try it actually.

Reply:
I tried it, it works as you intended, LGTM.